src/base/__tests__/Context.ts

```diff
@@ -826,17 +832,17 @@ describe('Context class', () => {
           dsMockUtils.createMockSecondaryKey({
             signer: signerIdentityId,
             permissions: dsMockUtils.createMockPermissions({
-              asset: [],
-              portfolio: [],
-              extrinsic: [],
+              asset: dsMockUtils.createMockAssetPermissions('Whole'),
+              extrinsic: dsMockUtils.createMockExtrinsicPermissions('Whole'),
+              portfolio: dsMockUtils.createMockPortfolioPermissions('Whole'),
             }),
           }),
           dsMockUtils.createMockSecondaryKey({
             signer: signerAccountId,
             permissions: dsMockUtils.createMockPermissions({
-              asset: null,
-              portfolio: null,
-              extrinsic: null,
+              asset: dsMockUtils.createMockAssetPermissions(),
+              extrinsic: dsMockUtils.createMockExtrinsicPermissions(),
+              portfolio: dsMockUtils.createMockPortfolioPermissions(),
```

Comment:
You should use `Whole` for these. Empty codecs are not a possibility in this case